src/uct/ib/dc/dc_mlx5.c

```diff
@@ -375,15 +375,18 @@ static ucs_status_t uct_dc_mlx5_iface_create_dci(uct_dc_mlx5_iface_t *iface,
 
 #if HAVE_DC_DV
 ucs_status_t uct_dc_mlx5_iface_dci_connect(uct_dc_mlx5_iface_t *iface,
-                                           uct_dc_dci_t *dci)
+                                           uint8_t ndci)
 {
     uct_ib_mlx5_md_t *md = ucs_derived_of(iface->super.super.super.super.md,
                                           uct_ib_mlx5_md_t);
+    uct_dc_dci_t *dci    = &iface->tx.dcis[ndci];
+    uint8_t lag_port     = uct_dc_mlx5_iface_dci_lag_port(iface, ndci);
```

Comment:
let's call it pool_index across the code, in case we would use dci pools for other things, not just lag